src/cclib/parser/data.py

```diff
@@ -75,8 +75,8 @@ class ccData(object):
         scftargets -- targets for convergence of the SCF (array[2])
         scfvalues -- current values for convergence of the SCF (list of arrays[2])
         temperature -- temperature used for Thermochemistry (float, kelvin)
-        transprop -- all absorption and emission spectra (dictionary {name:(etenergies, etoscs)})
-            WARNING: this attribute is not standardized and is liable to change in cclib 2.0
+        transprop -- all absorption and emission spectra (dictionary {name:(etenergies, etoscs)})
+            WARNING: this attribute is not standardized and is liable to change in cclib 2.0

```

Comment:
Don't change whitespace in an unrelated Pull Request.

Reply:
My Bad. Sorry about that.

Reply:
Am I looking at an outdated diff? It looks like you still have whitespace changes.